src/Asset.php

```diff
@@ -2,67 +2,86 @@
 
 namespace Acquia\ContentHubClient;
 
-class Asset extends \ArrayObject
-{
-    /**
-     *
-     * @param array $array
-     */
-    public function __construct(array $array = [])
-    {
-        parent::__construct($array);
-    }
+/**
+ * Class Asset.
+ *
+ * @package Acquia\ContentHubClient
+ */
+class Asset extends \ArrayObject {
 
-    /**
-     * @param string $url
-     *
-     * @return \Acquia\ContentHubClient\Asset
-     */
-    public function setUrl($url)
-    {
-        $this['url'] = $url;
+  /**
+   * Asset constructor.
+   *
+   * @param array $array
+   *   Asset data.
+   */
+  public function __construct(array $array = []) {
```

Comment:
So the { on the same line is a drupal standard, but seeing as how this is a php library we're going to follow the standard the rest of the php community uses which is { on a separate line. Let's change that back.